addons/binding/org.openhab.binding.enocean/README.md

```diff
@@ -0,0 +1,128 @@
+
+### Actuators
+ 
+If the actuator supports UTE teach-ins, the corresponding thing can be created and paired automatically. First you have to start the discovery scan for this binding in PaperUI. Then press the teach-in button of the actuator. 
+
+If the actuator does not support UTE teach-ins, you have to create, configure and choose the right EEP of the thing manually. It is important to link the teach-in channel of this thing. Afterwards you have to activate the pairing mode of the actuator. Then switch on the teach-in item(/channel) to send a teach-in message to the actuator. If the pairing was successful, you can control the actuator and unlink the teach-in channel.
+
+
+## Adding Enocean-Things (Actuators, Switches, Sensors, etc.)
+
+https://github.com/fruggy83/enocean/wiki#how-to-add-an-enocean-thing-actuator-sensor-wallswitch-etc-via-paperui
+
+## Thing Configuration
+
+Things can and should by configured through PaperUI. Following the most important config parameters:
+
+Bridge
+
+ * Serial port: The serial port to which the EnOcean gateway is connected to
+ * Next device id: The device Id which should be taken for the next created actuator without an explicit device Id
+ 
+Things
+
+ * Thing Id: EnOcean Id of the device. 4 byte hex string (abcdef00).
+ * Bridge: Must be provided
+ * Sender Id: Is used to generate the unique EnOcean device Id for sending messages (added to the base Id of the gateway). If you leave it empty, the next free Id will be determined automatically (see "next device id" of bridge). The resulting device Id can be seen through the properties.
+ * (Sending/receiving) EEP: Set the EEP which should be used to send and receive message to or from the device.
+ 
+## Generic Things
+
+  If an Enocean device uses an unsupported EEP or _A5-3F-7F_, you have to create a generice thing. Generic things support all generic channels like switch, number, string etc. However you have to specify how to convert the Enocean messages from the thing into Openhab state updates and how to convert the Openhab commands into Enocean messages. These conversion functions can be defined with the help of transformation functions like MAP or JavaScript.
+
+For an inbound transformation (Enocean message => Openhab state) you get the channel id and the Enocean data seperated by a pipe.
+
+```
+ChannelId|EnoceanData(Hex)=OpenhabState|Value
+genericLightSwitch|70=OnOffType|ON
+genericLightSwitch|50=OnOffType|OFF
+```
+
+Outbound transformation (Openhab command => Enocean message):
+
+```
+ChannelId|OpenhabCommand=EnoceanData(Hex)
+genericLightSwitch|ON=01000009
+genericLightSwitch|OFF=01000008
+```
+
+## Channels
+
+The channels of a thing are determined automatically based on the choosen EEP. The following channels are supported: (Light) Switch, Dimmer, Rollershutter, Temperature, Handle state and many more.
```

Comment:
Here as well: Please provide a table of all channes here and list, which thing type has which channels.

Reply:
fixed